perun-core/src/test/java/cz/metacentrum/perun/core/entry/AuditMessagesManagerEntryIntegrationTest.java

```diff
@@ -181,12 +182,21 @@ public void getMessagesPage_ascendingOrder() throws Exception {
 		perun.getAuditer().logWithoutTransaction(sess, new StringMessageEvent("Test 1"));
 		perun.getAuditer().logWithoutTransaction(sess, new StringMessageEvent("Test 2"));
 
-		MessagesPageQuery query = new MessagesPageQuery(2, 0, SortingOrder.ASCENDING);
+		MessagesPageQuery query = new MessagesPageQuery(2, 0, SortingOrder.ASCENDING, new ArrayList<>());
 
 		Paginated<AuditMessage> messages = perun.getAuditMessagesManager().getMessagesPage(sess, query);
 		assertThat(messages.getData().get(0).getId()).isLessThan(messages.getData().get(1).getId());
 	}
 
+	@Test
+	public void findAllPossibleEvents() throws Exception {
+		System.out.println(CLASS_NAME + "findAllPossibleEvents");
+
+		List<String> events = perun.getAuditMessagesManager().findAllPossibleEvents(sess);
+
+		assertThat(events.size()).isEqualTo(208);
```

Comment:
I think it would be pain to keep changing this test every time new audit event is created. Maybe just checking that some (>10?) are returned? @xflord ideas?

However, please add test case where you use the new query parameter to get audit messages (you can log some audit message with `perun.getAuditer().logWithoutTransaction` method)

Reply:
Larger than a set amount sounds reasonable. Also maybe check that it includes some of the essential events (MemberCreated for example), to make sure that the format of returned events is correct